agent-control/Cargo.toml

```diff
@@ -81,6 +81,9 @@ windows = { workspace = true, features = [
   "Win32_System_Threading",
 ] }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+windows-sys = "0.61.2"
+
 [dev-dependencies]
```

Comment:
We are also using `windows-sys` in resource-detection crate: https://github.com/newrelic/newrelic-agent-control/blob/main/resource-detection/Cargo.toml#L23

Should we move the dependency to the workspace's Cargo.toml?

Reply:
Sure!

Reply:
Done in last commit